delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/organizations/OrganizationsImplSpec.scala

```diff
@@ -40,7 +40,7 @@ class OrganizationsImplSpec
   val label2       = Label.unsafe("myorg2")
 
   private lazy val (scopeInitLog, orgs) = ScopeInitializationLog().map { scopeInitLog =>
-    scopeInitLog -> OrganizationsImpl(Set(scopeInitLog), config, xas, clock)
+    scopeInitLog -> OrganizationsImpl(ScopeInitializer.withoutErrorStore(Set(scopeInitLog)), config, xas, clock)
```

Comment:
We could get rid of `ScopeInitializationLog` and just make sure that `ScopeInitiliazer` has been called ?